src/functions/logisticLoss.jl

```diff
@@ -47,12 +47,12 @@ end
 #
 # Lipschitz constant of gradient: (mu y)
 
-function gradient!(g::AbstractArray{R}, f::LogisticLoss{T, R}, x::AbstractArray{R}) where {T, R}
+function gradient!(g::AbstractArray{R}, f::LogisticLoss, x::AbstractArray{R}) where R
     val = R(0)
```

Comment:
zero(R)

Reply:
I actually prefer `R(0)` to `zero(R)`, because the former can be used for any number

Reply:
yep, I imagine they're equivalent...